Makefile

```diff
@@ -133,7 +133,7 @@ perms-table:
 	@go run internal/website/permstable/permstable.go
 
 .PHONY: gen
-gen: cleangen proto api cli perms-table fmt copywrite
+gen: cleangen proto api cli perms-table fmt
```

Comment:
Notable change

Reply:
`make-gen-delta` is still failing? Is that expected?